guides/Using front-end libraries.md

```diff
@@ -0,0 +1,80 @@
+# Using front-end libraries
+
+Using Bower for manage javascript components.
+
+Install Bower: 
+
+```
+npm install -g bower
+```
+
+Create new `bower.json` file you can using `bower init`
+
+Now we can install packages with `bower install <package> --save`
+
+```
+bower install jquery --save
+```
+
+Use files only that you need while using Bower
```

Comment:
if we install packages without using bower-installer, bower copies entire repository of plugin into directory but we don't need it, we need only main files. Bower installer helps us to include only main files and moved them to the location that we want.